Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
 
+    void SetGeometry(GeometryTGeo *g) { fGeometry=g; }
```

Comment:
not sure if every chip needs this pointer object to get information from the geometry. Anyhow, presumably the geometry is fixed throughout the lifetime of the chip object, then it's better to set it up in the constructor